### ISSUE TITLE
USER STORY: Edit own task

### ISSUE THREAD

**Issue by blahosyl** [blahosyl/task-manager-frontend#16]
As a **Logged-in User**,

I want to **edit tasks owned by me or assigned to me**,

so that **update information as needed**

### Acceptance Criteria

1. The **Edit task** icon is only visible to users who own or are assigned to the task
2. Clicking the **Edit task** icon loads the TaskEditForm
3. The UI form displays all editable fields of a Task instance populated by the current values
4. The UI form validates user input
5. Clicking Submit updates the Task instance in the DB
6. The user receives visual confirmation of the updating of the task

### Tasks

- [x] add & style edit/delete dropdown
- [x] import `useHistory` (if not already done)
- [x] add `handleEdit` function:
  - redirect to `history.push(`/tasks/${id}/edit`)`
- [x] pass the `handleEdit` function to `MoreDropdown` (in `Task.js`)
- [x] write `TaskEditForm` based on TaskCreateForm #102 
- [x] import `TaskEditForm` to `App.js` & add route to it
- [x] check that task instance is updated on API
- [x] give visual confirmation to the user of editing the task: the updated TaskDetail page is shown

**Comment by blahosyl** [comment]
Blocked by #86

**Comment by blahosyl** [comment]
Blocked #102
